operatorcourier/validate.py

```diff
@@ -45,18 +45,19 @@ def _log_error(self, message, *args, **kwargs):
         self.validation_json['errors'].append(message % args)
         logger.error(message, *args, **kwargs)
 
-    def validate(self, bundle):
+    def validate(self, bundle, repository=None):
         """validate takes a bundle as a dictionary and returns a boolean value that
         describes if the bundle is valid. It also logs verification information when
         there are invalid sections in the bundle.
 
-        :param bundle: Dictionary of bundle value
+        :param bundle:     Dictionary of bundle value
```

Comment:
nit: Does this indent need to match for all params?

Reply:
@SamiSousa It seems that we don't have this indent in other places. I'll fix it.